tests/Avalonia.IntegrationTests.Appium/GestureTests.cs

```diff
@@ -118,12 +119,15 @@ public void DoubleTapped_Is_Raised_After_Control_Changes()
             // #8733
             var border = _session.FindElementByAccessibilityId("GestureBorder");
             var lastGesture = _session.FindElementByAccessibilityId("LastGesture");
-
+            
             new Actions(_session)
                 .MoveToElement(border)
                 .DoubleClick()
-                .DoubleClick()
```

Comment:
Now this is just doing a one double-click, i think it's no longer testing #8733?

Reply:
its doing double click... having to move to get around appium being shit... then double click again,